File: eTravelAgencija.Services/Database/OfferPlanDay.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace eTravelAgencija.Services.Database
{
    public class OfferPlanDay
    {
        [Key]
        public int Id { get; set; }

        [ForeignKey(nameof(OfferDetails))]
        public int OfferDetailsId { get; set; }

        public OfferDetails OfferDetails { get; set; }

        public int DayNumber { get; set; }

        public string DayTitle { get; set; }

        public string DayDescription { get; set; }
    }
}

File: UI/etravel_desktop/lib/models/offer_plan_day.dart
class OfferPlanDay {
  final int offerDetailsId;
  final String dayTitle;
  final String dayDescription;
  final int dayNumber;

  OfferPlanDay({
    required this.offerDetailsId,
    required this.dayTitle,
    required this.dayDescription,
    required this.dayNumber,
  });

  factory OfferPlanDay.fromJson(Map<String, dynamic> json) {
    return OfferPlanDay(
      offerDetailsId: json['offerDetailsId'],
      dayTitle: json['dayTitle'],
      dayDescription: json['dayDescription'],
      dayNumber: json['dayNumber'],
    );
  }
}
